### ISSUE TITLE
Add minimal adaptive CSS

### ISSUE THREAD

**Issue by toraritte** [toraritte/software-licensing-primer#1]
So that it renders ok on screens (one column of readable width) and on mobile devices (the column constraints do not mess up the view on a phone screen).

Currently it should be accessible for screen readers, so it's ok.

**Comment by toraritte** [comment]
https://internetingishard.com/html-and-css/responsive-design/

```css
      @media only screen and (max-width: 400px) {
        body {
          margin-left:  2%;
          margin-right: 2%;
        }
      }

      /* Tablet Styles */
      @media only screen and (min-width: 401px) and (max-width: 960px) {
        body {
          margin-left:  15%;
          margin-right: 15%;
        }
      }

      /* Desktop Styles */
      @media only screen and (min-width: 961px) {
        body {
          margin-left:  30%;
          margin-right: 30%;
        }
      }
```
